proto/buf.yaml

```diff
@@ -2,7 +2,7 @@ version: v1
 name: buf.build/conduitio/conduit
 lint:
   use:
-    - DEFAULT
+    - STANDARD
```

Comment:
For the record, `DEFAULT` is being deprecated so updating to use `STANDARD` instead.